backend/app/api/endpoints/openapi_responses.py

```
@@ -0,0 +1,550 @@
+                    status_code=status.HTTP_400_BAD_REQUEST,
+                    detail=f"Invalid previous_response_id format: '{request_body.previous_response_id}'",
+                )
+
+            # Verify previous task exists and belongs to the current user
+            existing_task = (
+                db.query(Kind)
+                .filter(
+                    Kind.id == previous_task_id,
+                    Kind.kind == "Task",
+                    Kind.user_id == current_user.id,
+                    Kind.is_active == True,
```

Ruff (0.14.8)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)